sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -424,6 +424,9 @@ object FunctionRegistry {
     expression[MakeInterval]("make_interval"),
     expression[DatePart]("date_part"),
     expression[Extract]("extract"),
+    expression[SecondsToTimestamp]("timestamp_seconds"),
+    expression[MilliSecondsToTimestamp]("timestamp_milliseconds"),
+    expression[MicroSecondsToTimestamp]("timestamp_microseconds"),
```

Comment:
I think it's okay to have expressions presumably to get away from `cast(ts as long)` behaviour which is not invasive.  But it would be more interesting to see how other DBMSes solved this problem. From arbitrary googling, vendors have different approaches.

Seems like BigQuery has these three functions (https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions?hl=ko#timestamp_seconds). Shall we match the naming at least?

Reply:
oh, I didn't realize it's `MILLIS` not `MILLISECONDS`. Let's follow the short name.

Reply:
@cloud-fan, is it means we shall rename `MilliSecondsToTimestamp` to `MillisToTimestamp` and `MicroSecondsToTimestamp` to `MicrosToTimestamp`.

Reply:
Not a big deal but yeah let's do that.